samples/javascript_nodejs/51.teams-messaging-extensions-action/index.js

```diff
@@ -59,3 +59,8 @@ server.post('/api/messages', (req, res) => {
         await bot.run(context);
     });
 });
+
```

Comment:
Again, I can't comment in the right place, but please fix the following errors (in `index.js`):

```cmd
  30:81  error  Trailing spaces not allowed                    no-trailing-spaces
  63:1   error  Expected space or tab after '//' in comment    spaced-comment
  66:5   error  Newline required at end of file but not found  eol-last
```

After making all of your changes please run `npm run lint` and ensure no other errors occur. Running `eslint . --fix` may also be helpful, although you may need to install eslint globally first, via `npm i -g eslint`